res/skins/LateNight/effect_unit.xml

```diff
@@ -93,11 +93,12 @@
                   <Text>ON</Text>
                 </State>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <ConfigKey persist="true"></ConfigKey>
                   <ButtonState>LeftButton</ButtonState>
                 </Connection>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <!-- FIXME: Quick hack until effect chains are implemented. -->
```

Comment:
The behavior with all effects connected to the button is erratic. It only lights up when Effect3 is enabled, regardless of the state of Effect1 and Effect2. I haven't quite figured out the rules determining how it behaves when clicking the button, but it's confusing. IMO leaving it as just connected to Effect1 is better for now.